ci/cloudbuild/builds/publish-docs.sh

```diff
@@ -21,14 +21,13 @@ source module ci/cloudbuild/builds/lib/cmake.sh
 source module ci/cloudbuild/builds/lib/features.sh
 
 mapfile -t FEATURE_LIST < <(features::list_full)
-read -r ENABLED_FEATURES < <(features::list_full_cmake)
 
 version=""
 doc_args=(
   "-DCMAKE_BUILD_TYPE=Debug"
   "-DGOOGLE_CLOUD_CPP_GENERATE_DOXYGEN=ON"
   "-DGOOGLE_CLOUD_CPP_GEN_DOCS_FOR_GOOGLEAPIS_DEV=ON"
-  "-DGOOGLE_CLOUD_CPP_ENABLE=${ENABLED_FEATURES}"
+  "-DGOOGLE_CLOUD_CPP_ENABLE=__ga_libraries__,__experimental_libraries__"
```

Comment:
Can you add: `experimental-opentelemetry` and `experimental-storage-grpc` back in?

I think you will lose documentation such as: https://googleapis.dev/cpp/google-cloud-storage/latest/namespacegoogle_1_1cloud_1_1storage__experimental.html

Reply:
Fixed